lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -153,7 +154,9 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
                 cohorts_status[cohort_name] = {
                     'Cohort Name': cohort_name,
                     'Students Added': 0,
-                    'Students Not Found': set()
+                    'Students Not Found': set(),
+                    'Invalid Email Addresses': set(),
+                    'Preregistered Users': 0
```

Comment:
One note about using "Preregistered", @catong, keep in mind that these learners are NOT actually registered in the course. When they do register for the course (or if they are preregistered for the course via the other instructor dashboard tool), then when they enter the course we will use this information to determine which cohort they should be in (instead of randomly assigning them to one).

So I'm not sure if there is a better word to use than "preregistered".